### PR TITLE
Check updates for github actions weekly

### PR DESCRIPTION
**Pull request by jlosito**
### What does this PR do?

This will cause dependabot to check for updates for any github action which this project uses on a weekly basis and submit pull requests with version bumps.

https://docs.github.com/en/code-security/dependabot/working-with-dependabot/keeping-your-actions-up-to-date-with-dependabot

### CODE REVIEW

**Comment by benoitf** [comment]
hi, thanks for the PR. but limiting dependabot checks to only github action is too restrictive

**Comment by jlosito** [comment]
Are you suggesting that this change should also include dependabot checking the javascript related dependencies as well? I can easily include that apart of this change if so.

**Comment by benoitf** [comment]
@jlosito probably yes

also for now I think dependabot is not enabled on repositories from containers

**Comment by benoitf** [comment]
also probably, we can go on a daily basis

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+version: 2
+updates:
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "weekly"
```
Comment (benoitf) [inline .github/dependabot.yml]: we should also limit the total of PRs

```suggestion
# Set update schedule for GitHub Actions

version: 2
updates:
  - package-ecosystem: "github-actions"
    directory: "/"
    schedule:
      interval: "daily"
    open-pull-requests-limit: 20

  - package-ecosystem: "npm"
    directory: "/"
    schedule:
      interval: daily
    open-pull-requests-limit: 20

```

**Comment by benoitf** [comment]
if you think there are other enhancements to apply, let's do it.
Else, LGTM

**Review by benoitf — APPROVED** [review]

**Comment by benoitf** [comment]
thanks @jlosito for your contribution

**Pull request merged**